h/templates/layouts/group.html.jinja2

```diff
@@ -9,13 +9,7 @@
 {% block content %}
   <div class="content paper">
     <div class="form-container">
-      {% if request.session.peek_flash('success') -%}
-        <div class="form-flash">
-          {% for message in request.session.pop_flash('success') %}
-            <p>{{ message }}</p>
-          {%- endfor %}
-        </div>
-      {%- endif %}
+      {% include "h:templates/includes/flash-messages.html.jinja2" %}
```

Comment:
This replaces existing flash message code with the new equivalent shared include, so it makes no difference other than removing duplication. I don't _think_ flash messages every do actually appear on `group.html.jinja2` though :man_shrugging:

Reply:
Good reuse